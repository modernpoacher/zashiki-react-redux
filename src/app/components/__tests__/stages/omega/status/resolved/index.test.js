import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

import Resolved from '@modernpoacher/zashiki-react-redux/app/components/stages/omega/status/resolved'

Enzyme.configure({ adapter: new Adapter() })

describe('@modernpoacher/zashiki-react-redux/app/components/stages/omega/status/resolved', () => {
  it('renders', () => {
    const component = (
      <Resolved
        description='MOCK DESCRIPTION'
        definition={{}}
        resource={{}}
        response={{}}
        errors={[]}
        onChange={jest.fn()}
        onSubmit={jest.fn()}
      />
    )

    expect(shallow(component))
      .toMatchSnapshot()
  })
})
