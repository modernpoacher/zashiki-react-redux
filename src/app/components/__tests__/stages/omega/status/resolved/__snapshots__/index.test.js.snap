// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@modernpoacher/zashiki-react-redux/app/components/stages/omega/status/resolved renders 1`] = `
<div
  className="omega resolved"
>
  <Omega
    definition={Object {}}
    errors={Array []}
    onChange={[MockFunction]}
    onSubmit={[MockFunction]}
    resource={Object {}}
    response={Object {}}
  />
  <withRouter(Connect(Component)) />
</div>
`;
